atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -157,7 +157,7 @@ def run(self):
             base_image_id = ""
 
         annotations = {
-            "dockerfile": self.get_pre_result(CpDockerfilePlugin.key),
+            "dockerfile": df_parser(self.workflow.builder.df_path, workflow=self.workflow).content,
```

Comment:
Could this be `open(self.workflow.builder.df_path).read()` instead? Would avoid parsing the file again.